scripts/data_guide/data_guide.qmd

```diff
@@ -134,7 +134,7 @@ AACR Project GENIE data versions follow a numbering scheme derived from [semanti
 
 AACR Project GENIE Data is currently available via two mechanisms:
 
--   Synapse Platform (Sage Bionetworks): https://synapse.org/genie
+-   Synapse Platform (Sage Bionetworks): https://genie.synapse.org/ or https://synapse.org/genie
```

Comment:
Should we just link to the portal?

Reply:
I think this is fine for now. Will probably want to do this next data guide update!